File: graph_distance_decay.jl
using RCall

R"""
library("ggplot2")
library("reshape2")
ind_effects = as.data.frame($(Array(ind_effects[:,:]))
colnames(ind_effects) = c("pool", "ninety", "eighty", "seventy", "sixty", "fifty", "fourty", "thirty", "twenty")
a = melt(ind_effects)
a$network = rep(1:145,9)
#a$distance = rep(c("pool", "ninety", "eighty", "seventy", "sixty", "fifty", "fourty", "thirty", "twenty"), each=28)
colnames(a) = c("distance", "value", "network")
ggplot(a, aes(x=factor(distance), y=value, group=factor(network))) +
  geom_line() +
  theme_classic()
"""

R"""
library("ggplot2")
library("reshape2")
ind_effects = as.data.frame($(Array(ind_effects)))
colnames(ind_effects) = c("pool", "ninety", "eighty", "seventy", "sixty", "fifty", "fourty", "thirty", "twenty")
a = melt(ind_effects)
#a$network = rep(1:28,9)
#a$distance = rep(c("pool", "ninety", "eighty", "seventy", "sixty", "fifty", "fourty", "thirty", "twenty"), each=28)
colnames(a) = c("network", "distance", "value")
ggplot(a, aes(x=factor(distance), y=value, group=factor(network))) +
  geom_line() +
  theme_classic()
"""

R"""
write.csv($(ind_effects[:,:]),file="resultado.csv", row.names=TRUE)
"""


convert(DataFrame, ind_effects[1,:,:]) 


CSV.write( "Teste_julia.csv",  ind_effects[1,:,:]);

File: VeryFirst_Exploratory_PreviousFirstCommiteeMeeting/graph_distance_decay.R

ind_effects = as.data.frame(ind_effects)
ind_effects$network = 1:59
colnames(ind_effects) = c("pool", "oit", "sess", "qua", "vint")

a = data.frame(rep(1:59,5))
  
a$values =  NA
a$distance = NA
a[1:59,2] = ind_effects[,1]
a[1:59,3] = "pool"
a[60:118,2] = ind_effects[,2]  
a[60:118,3] = "80"
a[119:177,2] = ind_effects[,3]                
a[119:177,3] = "60"
a[178:236,2] = ind_effects[,4] 
a[178:236,3] = "40"
a[237:295,2] = ind_effects[,5] 
a[237:295,3] = "20"

colnames(a) = c("network", "value", "distance")
a$distance = factor(a$distance, levels = c("pool", "80", "60", "40", "20"))

ggplot(a, aes(x=factor(distance), y=value, group=factor(network), color=factor(network)))+
  geom_line() +
  theme_classic()

a = result
